addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -294,6 +294,17 @@ $o-wblog-loader-size: 50px;
             display: none;
         }
     }
+
+}
+
+#oe_snippets > .o_we_customize_panel we-customizeblock-option.snippet-option-TagBlogPostSelect we-list we-row.pt-1 {
```

Comment:
can be probably simplified ;)
It was just an example that I provided to you to have an idea of the design.

Imo, it should be a new class depending of your  "isEditing"  and style only apply when this class is in dom